clash-lib/src/Clash/Netlist/BlackBox.hs

```diff
@@ -119,8 +119,10 @@ mkArgument bndr e = do
     let eTyMsg = "(" ++ showDoc e ++ " :: " ++ showDoc ty ++ ")"
     ((e',t,l),d) <- case hwTyM of
       Nothing   ->
-        return ((Identifier (error ($(curLoc) ++ "Forced to evaluate untranslatable type: " ++ eTyMsg)) Nothing
-                ,Void Nothing,False),[])
+        let identifier = case findRemovedArg e of
```

Comment:
```haskell
Nothing
  | (Prim nm _,_) <- collectArgs e
  , nm == "Clash.Transformations.removedArgs"
  -> return (Identifier (Text.fromString nm) Nothing, Void Nothing, False),[])
  | otherwise
  -> error ("Forced to evaluate untranslatable type: " ++ eTyMsg)
```